### DISCUSSION TITLE
Confusion about GPS DO LeoBodnar

### DISCUSSION THREAD

**Discussion by Gotsche43** [srsran/srsRAN_Project#573] [General Help] [answered]
As the headline says, I am confused about following... In my setup I have a Leo Bodnar GPS DO that is connected to my USRP B200mini ref channel.

In the gNB config file, I have put sync: external. But Im confused. Some gNB files says it should be clock: external. What are the right configuration if I use 10 MHz input from the gpsdo?

Please help a beginner

Regards

**Comment by pgawlowicz** [comment]
If you have the gpsdo module inside the your USRP, you can select clock and sync to be `gpsdo`.  But it seems that there is no option for gpsdo module for B200mini.

**Reply by Gotsche43** [reply]
I have it as an external input on the ref input. Maybe my question was unclear. Should I put sync or clock as external?

**Reply by pgawlowicz** [reply, marked as answer]
You are right, thanks for pointing it out, we need to unify our docs. 
You should set sync to external as in this example config file: https://github.com/srsran/srsRAN_Project/blob/main/configs/gnb_rf_b210_fdd_srsUE.yml

**Reply by Gotsche43** [reply]
Thank you!:-)
